argocd_examples/README.md

```diff
@@ -53,3 +61,26 @@ runtimeConf:
 ./get_satori_installation_package.sh
 ```
 
+## External secrets operator integration  
+
+1. If you have an [external secret operator](https://github.com/external-secrets/external-secrets) running on your K8S cluster, you can leverage it to dynamically create the kubernetes secret with the service key required for Satori DAC deployment.  
+2. Create a secret in your preferred secret manager (HasiCorp Vault, AWS secret manager, etc.). The secret could have any name, for this example we'll be using `satori-dac-service-key`
+3. Store the  `<SATORI_SERVICE_ID>` and `<SATORI_SERVICE_KEY>` in a json format  {"id": "<SATORI_SERVICE_ID>", "key": "<SATORI_SERVICE_KEY>"}  
+4. The helm values file should contains the following (the values might be different, depending on your external-secrets operator implementation): 
+
+```
+
+runtimeConf:
+  dacInstaller:
+    externalSecretOperator:
+      dacServiceAccountSecret: 
+        create: true
+        remoteRef:
+          key: `satori-dac-service-key`
+          property: ""
+      secretStoreRef:
+        name: "default-secret-store"
+        kind: "ClusterSecretStore"
+
+```
+5. Please ask Satori support for additional assistance.  
```

Comment:
Maybe something like "to enable this feature, please contact the Satori support team"